### ISSUE TITLE
Dota 2. CRASHED

### ISSUE THREAD

**Issue by Gogog686869** [ValveSoftware/Dota2-Gameplay#11101]
### Description

After starting Dota 2 and being in it for a couple of seconds, the game crashes and so on constantly. Any solutions don't work.Everything worked before the update and did not crash as it is now. Please fix your new patch with the launcher and rendering of the game

### Example Match ID (and possibly Timestamp)

before new patch with rendering and client

**Comment by SoloBerezin** [comment]
the same crap, I don't understand what to do, I re-installed the dota, checked the integrity of the files, rebooted the PC, nothing helps

**Comment by yng0dzz** [comment]
guys, I have same problem). Waiting for a patch fix, yep?
I hope we will have to wait only from 5 to 24 hours.. I WANNA TO FUCKING PLAY DOTA, WHY U STOPPING ME, VALVE????
That company knows how to break everything what worked well.
If u fix problem before new patch - mention me plz)00)9)

**Comment by ImaGaRoNiN** [comment]
Yes, the same game after one fix, nothing was fixed, well, it's a shame, I'll have to wait again, I hope that they will already fix it in the new fix
